pkg/registry/configmap.go

```diff
@@ -52,7 +52,7 @@ http:
 
 	viewerEnvfile := fmt.Sprintf(`
 ANALYTICS_WRITE_KEY=%s
-DEVFILE_REGISTRIES=[{\"name\":\"Community\",\"url\":\"http://localhost:8080\",\"fqdn\":\"http://%s.%s\"}]`,
+DEVFILE_REGISTRIES=[{"name":"Community","url":"http://localhost:8080","fqdn":"http://%s.%s"}]`,
```

Comment:
The `fqdn` format will not work on openshift deployments since the IngressDomain is only required for [Kubernetes deployment  ](https://github.com/devfile/registry-operator/blob/main/DEVFILE_REGISTRY.md#kubernetes) and it can be empty.  If a registry is successfully deployed it should [update the URL](https://github.com/devfile/registry-operator/blob/main/controllers/devfileregistry_controller.go#L159) in `cr.Status.URL` field so we can rely on that setting as long as it's not empty.